packages/qwik/src/optimizer/core/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ["Manu <manu@builder.io>"]
 name = "qwik-core"
 version = "0.2.0"
 edition = "2021"
-repository = "https://github.com/builder.io/qwik"
+repository = "https://github.com/builderio/qwik"
```

Comment:
Out of curiosity, why was this changed?

Reply:
Ohh it was a global replace. :-) But the old URL is 404 and new URL is the correct one.